publication-doi-commons/src/main/java/no/unit/nva/publication/doi/dynamodb/dao/DynamodbStreamRecordJsonPointers.java

```diff
@@ -78,6 +87,10 @@ public DynamodbStreamRecordJsonPointers(DynamodbImageType imageType) {
         String base = imageType.getValue();
         this.identifierJsonPointer = JsonPointer.compile(base + IDENTIFIER);
         this.statusJoinPointer = JsonPointer.compile(base + STATUS);
+        this.doiRequestJsonPointer = JsonPointer.compile(base + DOI_REQUEST);
+        this.doiRequestStatusJsonPointer = JsonPointer.compile(DOI_REQUEST_STATUS);
```

Comment:
This is missing base?

Reply:
As I assume DoiRequest resides either under `oldImage` or `newImage`?

Reply:
They are using doiRequest as rootNode. I had to do it this way to make them work in PublicationMapper

Reply:
@janbj :  Yes I noticed! It's because `DynamodbStreamRecordImageDao.doiRequest` is a JsonNode.